README.md

```diff
@@ -1,9 +1,31 @@
 # Calculater
 
-a funny and tricky calculator
+A calculator with the basic functions.
```

Comment:
it is more than a basic calculator . 
it was made for trick basically 
type 1501 and click on menu button and explore additional settings
click on Advance Settings and create a key value pair 
key must be 4 digits long 
value must be 8 digits long 

now the trick begins 
as you type the key and press '='
then as you begin doing calculations 
after 4 = signs pressed calculator will show the result "value you fixed in advance settings"

watch this video 
this calculator works the same 
https://www.youtube.com/watch?v=hNkF7I1K8oo